sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DataSourceV2Strategy.scala

```diff
@@ -245,6 +246,28 @@ object DataSourceV2Strategy extends Strategy with PredicateHelper {
       }.toArray
       DeleteFromTableExec(r.table.asDeletable, filters) :: Nil
 
+    case UpdateTable(r: DataSourceV2Relation, attrs, values, condition) =>
+      val nested = attrs.asInstanceOf[Seq[Any]].filterNot(_.isInstanceOf[AttributeReference])
+      if (nested.nonEmpty) {
+        throw new AnalysisException(s"Update only support non-nested fields. Nested: $nested")
+      }
+      val attrsNames = DataSourceStrategy.normalizeFilters(attrs, r.output)
```

Comment:
I think we should rename `normalizeFilters` to `normalizeAttributesNames`

Reply:
Done.